lib/datura/data_manager.rb

```diff
@@ -203,11 +198,28 @@ def prepare_files
     return file_classes
   end
 
+    def prepare_xslt
+    # check modification date of gemfile.lock against the hidden script files
+    # if gemfile newer, copy the xslt over into the hidden files
+    gflock = File.join(@options["collection_dir"], "Gemfile.lock")
+    xslt = File.join(@options["collection_dir"], "scripts", ".xslt")
```

Comment:
I made it hidden because I'm afraid that XSLT savvy users (Nikki, Kevin, Andy, etc) will try to edit those files instead of using the override template system, but then of course their changes will be blown away by the next gem update!  So my solution was simply to put it in a directory that they might not be able to see very easily and then .gitignore it.  I am not averse to changing the name of this file, although unfortunately since I already started converting data repos we'll have to remember to change the references across a couple repositories and check them again.  We'll also want to come up with a way to avoid users changing the files without realizing that their changes are temporary.

What do you mean the name?  The name of the directory that the base XSLT is copied into?

Reply:
At this point, I like the hidden file solution, unless we think of anything better. I understand Greg's concerns, and maybe we can think of a better solution down the road, but I do think it's likely people will start editing XSLT if there's easy access. 

I also remembering that this is what we agreed to when we met to talk about this. 

I don't understand @techgique's  "name" comment either.

Reply:
Okay, yeah I think you're right that if we don't hide it, it will confuse folks. I hadn't thought about that, or forgot past discussion. Sorry for the confusion on the "name" comment. I should have added the comment on the line below this one where `datura_xslt` is the name of a variable. I mean to suggest changing it to `.datura-xslt` or `.xslt-datura`

Reply:
Sounds good to me! I'm hopeful we can come up with a better solution eventually. Like maybe a substitute for XSLT haha.